### PR TITLE
[COLLECTIONS-760]: Add tests for MapUtils

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**

### CODE REVIEW

File: src/test/java/org/apache/commons/collections4/MapUtilsTest.java [hunk 1]
```diff
@@ -56,6 +58,7 @@
  */
 @SuppressWarnings("boxing")
 public class MapUtilsTest extends AbstractAvailableLocalesTest {
+    private static final String THREE = "Three";
```
Comment (kinow) [inline src/test/java/org/apache/commons/collections4/MapUtilsTest.java:61]: Hi @Isira-Seneviratne should we have constants for "Two", and "key"? As this is a test class, I think we can have it either way, with or without constants, as the class size is not huge as well. Up to you :+1:

Comment (Isira-Seneviratne) [reply src/test/java/org/apache/commons/collections4/MapUtilsTest.java:61]: Assigning "Three" to a constant variable was recommended by SonarLint. It did not provide the same suggestion for "Two" and "key" as those are shorter strings.

Comment (Isira-Seneviratne) [reply src/test/java/org/apache/commons/collections4/MapUtilsTest.java:61]: On that note, maybe a configuration for [SonarQube](https://www.sonarqube.org/) code analysis could be added?

Comment (kinow) [reply src/test/java/org/apache/commons/collections4/MapUtilsTest.java:61]: I believe we were using it at one point. ASF Infra has or had a SonarQube instance running. Now I think there are projects in the ASF using SonarQube, though I am not sure whether that's SonarQube hosted within ASF, or the SonarCloud cloud service. Feel free to reach out via the dev-list and ask it there (as not all Apache Commons developers are subscribed to GitHub, the mailing lists are the best way to discuss things like this :)

**Pull request closed without merging**